external/akka/src/main/scala/org/apache/spark/streaming/akka/ActorReceiver.scala

```diff
@@ -90,6 +90,48 @@ object ActorReceiver {
 @DeveloperApi
 abstract class ActorReceiver extends Actor {
 
+  /**
+   * start a new actor-based receiver, spark streaming will asynchronously send the corresponding
+   * ActorRef object to the caller actor of this API
+   *
+   * @param props the props object of the receiver actor
+   * @param name  the name of the actor
+   */
+  def startNewActorReceiver(props: Props, name: String) {
```

Comment:
One of the options might be that we make all these APIs as blocking, since all of these are kind of control messages....